dash_cytoscape/utils/Tree.py

```diff
@@ -0,0 +1,158 @@
+from collections import deque
+
+
+class Tree(object):
+    def __init__(self, node_id, children=None, data=None, edge_data=None):
+        """
+        A class to facilitate tree manipulation in Cytoscape
+        :param node_id: The ID of this tree, passed to the node data dict
+        :param children: The children of this tree, also Tree objects
+        :param data: Dictionary passed to this tree's node data dict
+        :param edge_data: Dictionary passed to the data dict of the edge connecting this tree to its
+        parent
+        """
+        self.node_id = node_id
+        self.edge_data = edge_data
+        self.data = data
+        self.children = children
+
+        self.index = {}
+
+        if not children:
```

Comment:
Somehow it would seem more natural to me to have these three `if`s at the very top of the function, and assign `self.data = data` etc. afterwards.

Reply:
Doen